### ISSUE TITLE
Copy-button for copying stacktrace of failed test in Gradle-generated HTML test reports

### ISSUE THREAD

**Issue by gavvvr** [gradle/gradle#30449]
### Expected Behavior

Hello. It would be great to have a copy-button for copying a stack trace of a failed test to clipboard, when going through HTML test report generated by Gradle.

### Context

Quite often my flow is to:
1) execute all test suites with a single lifecycle Gradle task from terminal (instead of clicking buttons in Intellij) and get an aggregated HTML report
2) open this local aggregated report in the browser in case of failures
3) copy stack traces and paste it into Intellij's "Analyze Stack Trace or Thread Dump ..." tool for quick navigation to a failed test and exploring stack trace

Would be great to have a "Copy button" instead of using a mouse cursor and scrolling if stacktrace doesn't fit the screen.

**Comment by octylFractal** [comment]
I don't think this is major enough to warrant a release note. It's easily discovered when viewing the report.
